### PR TITLE
Upgrade gnss resources

### PR DESCRIPTION
**Pull request by frigusgulo**

### CODE REVIEW

File: src/es_sfgtools/processing/operations/gnss_resources.py [hunk 1]
```diff
@@ -24,18 +26,261 @@ def _parse_date(date:datetime.date | datetime.datetime)-> Tuple[str,str]:
     elif doy < 100:
         doy = f"0{doy}"
```
Comment (mikegottlieb84) [inline src/es_sfgtools/processing/operations/gnss_resources.py:27]: what if doy<10?  wouldn't you need 2 leading zeros?

File: src/es_sfgtools/processing/operations/pride_utils.py [hunk 2]
```diff
@@ -96,6 +165,42 @@ def get_daily_rinex_url(date:datetime.date) ->Dict[str,Dict[str,RemoteResource]]
...
+                    }
+            }
+    Note:
+        Until the remote resorces are updated with 'update_source', the file_name attribute will be None.
+    """
+    
+
     urls = {
         "sp3": {
             "wuhan": WuhanIGS.get_product_sp3(date),
```
Comment (mikegottlieb84) [inline src/es_sfgtools/processing/operations/pride_utils.py:206]: is there a reason to try Wuhan before France? my intuition would be to try France then Wuhan, as the download (if they have it) from France should be much faster...

**Review by mikegottlieb84 — COMMENTED** [review]
looks a lot cleaner, glad to hear it works!  a couple small comments but otherwise I think this looks great!

File: src/es_sfgtools/processing/operations/gnss_resources.py (hunk 1 above)
Comment (frigusgulo) [reply src/es_sfgtools/processing/operations/gnss_resources.py:27]: that is handled in line 24

File: src/es_sfgtools/processing/operations/pride_utils.py (hunk 2 above)
Comment (frigusgulo) [reply src/es_sfgtools/processing/operations/pride_utils.py]: good point. I swapped those

**Pull request merged**